gto-support-picasso/src/main/java/org/ccci/gto/android/common/picasso/widget/TextViewDrawableEndTarget.kt

```diff
@@ -0,0 +1,23 @@
+package org.ccci.gto.android.common.picasso.widget
+
+import android.graphics.drawable.Drawable
+import android.widget.TextView
+import org.ccci.gto.android.common.picasso.BaseViewTarget
+import org.ccci.gto.android.common.picasso.R
+
+class TextViewDrawableEndTarget(view: TextView) : BaseViewTarget<TextView>(view) {
+    init {
+        view.setTag(R.id.picasso_textViewDrawableEndTarget, this)
+    }
+
+    companion object {
+        fun of(view: TextView) = view.getTag(R.id.picasso_textViewDrawableEndTarget) as? TextViewDrawableEndTarget
+                ?: TextViewDrawableEndTarget(view)
+    }
+
+    public override fun updateDrawable(drawable: Drawable?) {
+        val current = view.compoundDrawablesRelative
+        drawable?.apply { setBounds(0, 0, intrinsicWidth, intrinsicHeight) }
+        view.setCompoundDrawablesRelative(drawable, current[1], current[2], current[3])
```

Comment:
you are setting `drawable` to the start drawable here, it should replace the end drawable

Reply:
So it needs to be set to ```current``` instead then as that is referencing the end drawable?